### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by aaronpalermo**
Replaced 'mange' with 'manage' in several files.

### CODE REVIEW

**Comment by thomascellerier** [comment]
Looks good, you need to sign the CLA first before we can merge.

**Review by kajes — CHANGES_REQUESTED** [review]
Don't merge this now. The html-files are generated, so the typos should be fixed in the source.

**Review by kajes — APPROVED** [review]
Ok, just did an extra check, and this seems to change just the things that would be generated anyway, so good to merge :+1: .

**Pull request merged**
